doc/getting-started.md

```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
 * [To run](#to-run)
 * [To initialize](#to-initialize)
 * [To use](#to-use)
-* [Integrating into a FOLIO-backend Vagrant box](#integrating-into-a-folio-backend-vagrant-box)
-* [To unassociate, undeploy and remove a running module](#to-unassociate-undeploy-and-remove-a-running-module)
-* [Adding permissions on the UI side](#adding-permissions-on-the-ui-side)
 
 ## Prerequisites
 
 You need
 [yaz4j](https://github.com/indexdata/yaz4j)
 and
 [YAZ](https://www.indexdata.com/yaz)
-before you can compile and install mod-copycat. In the directory next to your mod-copycat checkout, install as follows:
+before you can compile and install mod-copycat.
+
+Installing YAZ is easy on Ubuntu:
+
+	term1$ sudo apt install libyaz5
+
+yaz4j is distributed in https://maven.indexdata.com APT repo.
+If you are using Ubuntu, it is not necessary to compile and install yaz4j.
+
+Steps for install yaz4j (not necessary on Ubuntu amd64).
+In the directory next to mod-copycat checkout, install as follows:
 
 	term1$ sudo apt install yaz libyaz-dev swig
 	term1$ git clone https://github.com/indexdata/yaz4j.git
 	term1$ cd yaz4j
 	term1$ git checkout v1.6.0
 	term1$ mvn -B -Pbundle install
 
-This installs `yaz4j-1.6.0-SNAPSHOT.jar` into the local Maven repository.
+This installs `yaz4j-1.6.0.jar` into the local Maven repository.
 
-Because we used the `bundle` profile, the jar file includes a shared object. If running on same platform as we compile, that's fine. If not, you'll have to ship a shared object separately and install it in the path. The shared object is installed in `target/native` in all cases (whether bundle is specified or not).
+Because `bundle` profile was specified, the jar file includes a shared object.
+If running on same platform, that's fine.
+If not, you'll have to ship a shared object separately and install it in the path.
+The shared object is installed in `target/native` in all cases (whether bundle is specified or not).
 
 ## To build
 
 Once the yaz4j library is available, mod-copycat itself can be built:
 
-	term1$ cd ../mod-copycat
+	term1$ cd mod-copycat
 	term1$ mvn install
 
 ## To run
 
-	term1$ java -jar target/mod-copycat-fat.jar
+You must have PostgresQL running. Set environment appropriately:
+
+	$ export DB_HOST=localhost
+	$ export DB_PORT=5432
+	$ export DB_USERNAME=username
+	$ export DB_PASSWORD=password
+	$ export DB_DATABASE=database
+
+With these set, you can run:
+
+	$ java -jar target/mod-copycat-fat.jar
 
 ## To initialize
 
 In another terminal:
 
-	term2$ curl -d'{"module_to":"foo"}' -HX-Okapi-Tenant:testlib "-HAccept:*/*" -HContent-Type:application/json http://localhost:8081/_/tenant
-	[]
-	term2$ 
+	$ curl -d'{"module_to":"foo","parameters":[{"key":"loadReference","value":"true"}]}' \
```

Comment:
What does the "foo" mean here? I don't see any mention of mod-copycat in the invocation, so how does Okapi know what module to initialize? Could it be that we meant to say "mod-copycat" and accidentally said "foo" instead?

Reply:
Okapi is not involved here. And the module doesn't use this.

Reply:
Oh, stupid of me to have assumed Okapi!